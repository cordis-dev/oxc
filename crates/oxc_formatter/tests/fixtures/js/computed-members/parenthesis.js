new (get(win))[ty](xxx);
new (get(win))[ty][ty](xxx);

new (A?.[ty])(xxx);
new (A?.B[ty])(xxx);

new window['TouchEvent'](xxx);
