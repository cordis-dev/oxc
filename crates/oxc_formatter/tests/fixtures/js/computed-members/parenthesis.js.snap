---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
new (get(win))[ty](xxx);
new (get(win))[ty][ty](xxx);

new (A?.[ty])(xxx);
new (A?.B[ty])(xxx);

new window['TouchEvent'](xxx);

==================== Output ====================
new (get(win)[ty])(xxx);
new (get(win)[ty][ty])(xxx);

new (A?.[ty])(xxx);
new (A?.B[ty])(xxx);

new window["TouchEvent"](xxx);

===================== End =====================
